### PR TITLE
Add a documentation page for ethereum-types package

### PR DESCRIPTION
**Pull request by LogvinovLeon**
## Description

Adds a docs page for ethereum-types package

## Motivation and Context

Other people that will build ethereum-related type definitions are gonna use those and they'd better be documented.

## How Has This Been Tested?

Manually
screen shot 2018-06-04 at 11 02 09

## Types of changes

*   [ ] Bug fix (non-breaking change which fixes an issue)
*   [x] New feature (non-breaking change which adds functionality)
*   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

*   [ ] Change requires a change to the documentation.
*   [ ] Added tests to cover my changes.
*   [ ] Added new entries to the relevant CHANGELOG.jsons.
*   [x] Labeled this PR with the 'WIP' label if it is a work in progress.
*   [x] Labeled this PR with the labels corresponding to the changed package.

### CODE REVIEW

File: packages/website/ts/containers/ethereum_types_documentation.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            'ConstructorStateMutability',
+            'StateMutability',
+            'MethodAbi',
+            'ConstructorAbi',
+            'FallbackAbi',
+            'EventParameter',
+            'EventAbi',
+            'DataItem',
+            'OpCode',
+            // 'StructLog',
```
Comment (fabioberger) [inline packages/website/ts/containers/ethereum_types_documentation.ts]: Should this still be commented out?

**Comment by LogvinovLeon** [comment]
I did add it to the menu. It shows up on the menu for me.

File: packages/website/ts/containers/ethereum_types_documentation.ts (hunk 1 above)
Comment (LogvinovLeon) [reply packages/website/ts/containers/ethereum_types_documentation.ts]: Yes. There is an issue with the docs and they break when trying to render that type. I'll add it as a TODO

File: packages/website/ts/components/top_bar/top_bar.tsx [hunk 2]
```diff
@@ -165,6 +165,12 @@ export class TopBar extends React.Component<TopBarProps, TopBarState> {
                     primaryText={this.props.translate.get(Key.SolCov, Deco.CapWords)}
                 />
             </Link>,
+            <Link key="subMenuItem-ethereum-types" to={WebsitePaths.EthereumTypes} className="text-decoration-none">
+                <MenuItem
+                    style={{ fontSize: styles.menuItem.fontSize }}
+                    primaryText={this.props.translate.get(Key.EthereumTypes, Deco.CapWords)}
+                />
+            </Link>,
```
Comment (fabioberger) [inline packages/website/ts/components/top_bar/top_bar.tsx]: You also need to add this to the `_renderDrawer` method below or else it won't show up on the mobile menu.

**Pull request merged**